actions/delete.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/**
+ * Default entity delete action
+ *
+ * @package Elgg
+ * @subpackage Core
+ */
+$guid = get_input('guid');
+$entity = get_entity($guid);
+
+if (!$entity instanceof ElggEntity) {
+	register_error(elgg_echo('entity:delete:item_not_found'));
+	forward(REFERRER);
+}
+
+if (!$entity->canDelete()) {
+	register_error(elgg_echo('entity:delete:permission_denied'));
+	forward(REFERRER);
+}
+
+set_time_limit(0);
+
+// determine what name to show on success
+$display_name = $entity->getDisplayName();
+if (!$display_name) {
+	$display_name = elgg_echo('entity:delete:item');
+}
+
+$type = $entity->getType();
+$subtype = $entity->getSubtype();
+$container = $entity->getContainerEntity();
+
+if ($entity->delete()) {
+	// determine forward URL
+	$forward_url = get_input('forward_url');
+	if (!$forward_url) {
+		$forward_url = REFERRER;
+		$referrer_url = !empty($_SERVER['HTTP_REFERER']) ? $_SERVER['HTTP_REFERER'] : '';
+		if ($referrer_url) {
+			$segments = explode('/', parse_url($referrer_url, PHP_URL_PATH));
```

Comment:
@mrclay Is there a way to get these without having to parse them? This might cause issues in edge cases when Elgg is installed in a subdirectory with a numeric value